cloudify-manager-worker/templates/statefulset.yaml

```diff
@@ -166,6 +288,9 @@ spec:
         {{- end }}
         {{- end }}
       volumes:
+      - name: applogs
+        hostPath:
+          path: /tmp/log      
```

Comment:
You will have a problems there because this folder is not unique. If several cloudily-manader-worker stacks will be installed on the same node all of them will mount the same volume /tmp/log from k8s worker node and then rsync process from the different containers will sync logs inside  this folder erasing each other's changes.
In theory you can avoid this problem if you will use combination of namespace name in hostPath.path (something like "/tmp/log/{{.Release.Namespace}}") + pod name in rsync script (this is for avoid issues if cloudily-manader-worker stack launched in HA mode with 2 replicas in statefulset).
But again - if it's possible to avoid use rsync logs to temporary place we should do it.